tracer/test/test-applications/Samples.Shared/SampleHelpers.cs

```diff
@@ -72,6 +74,20 @@ public static void ConfigureTracer(string serviceName)
             ConfigureMethod.Invoke(tracer, new object[] { tracerSettings });
         }
 
+        public static void ConfigureTracerWithLogsInjection()
```

Comment:
nit: It looks like this and the `SetLogsInjectionEnabled` MethodInfo are no longer used after your revisions